docs/source/guide/core-concepts.myst

```diff
@@ -7,4 +7,5 @@ maxdepth: 1
 frontends-backends.myst
 executors.myst
 observables.myst
+calibrators
```

Comment:
[Optional]
Unless it creates problems, you can rename the file from `calibrators.md` to `calibrators.myst`. After this change, you can use its full extension in the core-concepts page instead of just `calibrators`.

Reply:
I have a slight preference to have the files named `.md` since most editors will allow some sort of rendering capability that is not available when the file is named `.myst`. I've thought about suggesting we rename myst files to md for this reason before, but just never got around to it. I'll put it on the agenda for the next mitiq meeting.

In the meantime, if you'd prefer to add the extension `calibrators.md` to this core concepts page, i'm happy to do that! WDYT?

Reply:
Fine to keep `.md` in this PR.  Good to discuss in a Mitiq meeting.